### DISCUSSION TITLE
BSD binaries

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [meilisearch/product#656] [Feedback & Feature Proposal]
Since there are already distributed macOS, Linux, and Windows binaries, including binaries for FreeBSD and OpenBSD would be helpful :)

**Comment by curquiza** [comment]
Hello @ThatOneCalculator 

For anyone reading this issue: the CI publishing binaries are here -> https://github.com/meilisearch/meilisearch/blob/main/.github/workflows/publish-binaries.yml
You can suggest opening PRs that update the CI so that you see the FreeBSD and OpenBSD binaries in a future release. We would be happy to review your PR!
However, the maintenance will be done by the community, except if a lot of people use it: we indeed try to focus our time on the most used binaries. But the open-source community will always be welcome to improve it! 😊 

As a workaround, you can compile from source files: [check out this guide](https://www.meilisearch.com/docs/learn/getting_started/installation), in the `source` tab
